UI/volume-control.cpp

```diff
@@ -594,10 +617,12 @@ VolumeMeter::VolumeMeter(QWidget *parent, obs_volmeter_t *obs_volmeter,
 	magnitudeIntegrationTime = 0.3;                  //  99% in 300 ms
 	peakHoldDuration = 20.0;                         //  20 seconds
 	inputPeakHoldDuration = 1.0;                     //  1 second
+	meterThickness = 3;     // Bar thickness in pixels
+	meterFontScaling = 0.8; // Font size is 80% of Widget font-size
```

Comment:
Pick "Font size" or "font-size".

Reply:
"font-size" is the item key in the QSS which can set the "font size" used by the QWidget. Using "font-size" for the first two words of the comment would be incorrect: it ISN'T a QSS value. I used "font-size" for the end of the comment to indicate that you can override the font via the QSS. I should have said QWidget instead of Widget, however.

Reply:
If they are actually referring to different things, then perhaps the comment requires further clarification.

> Font size is 80% of QWidget's font-size QSS property

Or some such.